qa/L0_perf_analyzer/test.sh

```diff
@@ -662,6 +662,19 @@ for PROTOCOL in grpc http; do
     fi
     curl localhost:8000/v2/trace/setting -d '{"trace_level":["OFF"]}'
     set -e
+
+    # Testing that setting trace file does not work
+    $PERF_ANALYZER -v -i $PROTOCOL -m simple_savedmodel_sequence_object \
+    -p 2000 -t5 --sync --trace-file trace.json --trace-level TIMESTAMPS \
+    --trace-rate 1000 --trace-count 100 --log-frequency 10 \
+    --input-data=$SEQ_JSONDATAFILE -s ${STABILITY_THRESHOLD} >$CLIENT_LOG 2>&1
```

Comment:
nit: I'd remove all unnecessary args other than `--trace-file` and the `-m model` if they're irrelevant to the test, since we're just expecting it to fail at parsing/startup.

Reply:
Good note, I wasn't sure, but upon further reflection, it seems like they shouldn't matter. I left `-v` as that's a pattern and I also left setting the network protocol as I want to test both lower level clients send the trace file option correctly and receive the error correctly.